source/plugin/blocks/accessing.rst

```diff
@@ -2,217 +2,150 @@
+    import org.spongepowered.api.world.server.ServerWorld;
+    import org.spongepowered.api.ResourceKey;
 
-    public Location<World> getBlockAt(String worldName, int posX, int posY, int posZ) {
-        World world = Sponge.getServer().getWorld(worldName).get();
-        Location<World> blockLoc = new Location<World>(world, posX, posY, posZ);
+    public ServerLocation getBlockAt(ResourceKey worldKey, int posX, int posY, int posZ) {
+        ServerWorld world = Sponge.server().worldManager().world(worldKey).get();
+        ServerLocation blockLocation = world.location(posX, posY, posZ);
         return blockLoc;
     }
 
 .. warning::
 
-    Note that the above example does not check if the world exists. ``getWorld(worldName).get()`` will fail if there
+    Note that the above example does not check if the world exists. ``world(worldKey).get()`` will fail if there
     is no world of that name loaded.
 
 
-With this ``Location`` object you can then obtain further information about the block. The following code checks if a
+With this ``ServerLocation`` object you can then obtain further information about the block. The following code checks if a
 referenced block is any kind of banner by checking the blocks type.
 
 .. code-block:: java
 
     import org.spongepowered.api.block.BlockType;
     import org.spongepowered.api.block.BlockTypes;
+    import org.spongepowered.api.tag.BlockTypeTag;
+    import org.spongepowered.api.world.server.ServerLocation;
 
-    public boolean isBanner(Location<World> blockLoc) {
-        BlockType type = blockLoc.getBlock().getType();
-        return type.equals(BlockTypes.STANDING_BANNER)
-                || type.equals(BlockTypes.WALL_BANNER);
+    public boolean isBanner(ServerLocation blockLoc) {
+        Tag<BlockType> bannerTag = BlockTypeTags.BANNERS.get();
+        BlockType blockType = blockLoc.blockType();
+        return bannerTag.contains(blockType);
     }
 
 .. tip::
     
-    The function ``==`` could be used in place of ``equals()`` as there is only one :javadoc:`BlockType` instance for
+    The function ``==`` could be used in place of ``equals()`` when it comes to comparing two :javadoc:`BlockType`'s as there are a fixed number of instance for
     every block, however it is generally recommended to use ``equals()``.
 
 Block Data Manipulators
 ~~~~~~~~~~~~~~~~~~~~~~~
 
 The data of a block is held as a :javadoc:`DataManipulator`, similar to other parts of the API. This is the container
-that holds information on components of our block such as the orientation of a block, specific types (stone vs.
-granite), and so on. Checking the values of these manipulators is easy, you just need to check the block's direction
-:javadoc:`DirectionalData`.
+that holds information on components of our block such as the orientation of a block, the blocks light level, and so on. Checking the values of these manipulators is easy, you just need to check the block's direction
```

Comment:
Line length <120 please. 
This bit could also improve by being a bit shorter anyway; "the orientation of a block, the blocks light level, and so on." could be "the orientation, light level, and so on." (Implicitly they belong to our block).

Reply:
Removed ``block`` from the wording, also ensured line length